File: doc/src/scripts/InfernalRobotics.lua
local krpc = require 'krpc.init'
local platform = require 'krpc.platform'
local Types = require 'krpc.types'

local conn = krpc.connect(nil, nil, nil, 'InfernalRobotics Example')

local group = conn.infernal_robotics.servo_group_with_name('MyGroup')
if group == Types.none then
  print('Group not found')
  os.exit(1)
end

for _,servo in ipairs(group.servos) do
  print(servo.name, servo.position)
end

group:move_right()
platform.sleep(1)
group:stop()

File: doc/src/scripts/InfernalRobotics.py
import krpc, time
conn = krpc.connect(name='InfernalRobotics Example')

group = conn.infernal_robotics.servo_group_with_name('MyGroup')
if group is None:
    print('Group not found')
    exit(1)

for servo in group.servos:
    print servo.name, servo.position

group.move_right()
time.sleep(1)
group.stop()
